plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/expression/VariableEmbedding.kt

```diff
@@ -66,7 +67,9 @@ class AnonymousVariableEmbedding(n: Int, override val type: TypeEmbedding) : Var
  * Embedding of a variable that comes from some FIR element.
  */
 class FirVariableEmbedding(override val name: MangledName, override val type: TypeEmbedding, val symbol: FirBasedSymbol<*>) :
-    VariableEmbedding
+    VariableEmbedding {
+    override fun toViper(source: KtSourceElement?): Exp.LocalVar = Exp.LocalVar(name, type.viperType, source.asPosition, symbol.asInfo)
```

Comment:
For clarity, I would recommend keeping `toViper` the same in all implementations of `VariableEmbedding` and providing an `info` property instead.